src/common/MuxConfig.h

```diff
@@ -387,6 +396,8 @@ class MuxConfig
     bool mEnableSwitchoverMeasurement = false;
     uint32_t mDecreasedTimeoutIpv4_msec = 10;
 
+    uint32_t mMuxReconciliationTimeout_sec = 10;
```

Comment:
This may be a repeated question and may be answered already:

Should this timeout not be the function of number of ports on the device? For example, 28 ports vs 128 ports may make some considerable difference.

Reply:
I didn't see much time differences between 56 ports and 128 ports (both took around 6 seconds), think we should be good here.